lib/wallaroo/core/checkpoint/checkpoint_initiator_phase.pony

```diff
@@ -95,6 +95,16 @@ class _WaitingCheckpointInitiatorPhase is _CheckpointInitiatorPhase
   fun ref resume_checkpointing_from_rollback() =>
     None
 
+  fun ref event_log_checkpoint_complete(worker: WorkerName,
```

Comment:
I address this in a more general fashion in c12a436a5c2e4c9167d9f2a35518a6a74449cc56

Reply:
Removed in a8a893069

Reply:
Removed by a8a893069